src-tauri/Cargo.toml

```
@@ -29,7 +29,7 @@ tokio = { version = "1", features = ["full"] }
 sentry = "0.30"
 sentry-log = "0.30.0"
 # Find steam games
-steamlocate = "1.0.2"
+steamlocate = "1.2.*"
```

Bit nit-picky but the recommended syntax to just pick newest patch level is just to omit it, i.e.

```suggestion
steamlocate = "1.2"
```

---

Adding fix myself